### PR TITLE
add contact route, page and link in footer

### PR DESCRIPTION
**Pull request by lesliekimm**
Really simple contact us page for now

### CODE REVIEW

File: app/views/static_pages/contact.html.erb [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+<div class="content-wrap">
+  <div class="container footer-margin">
+    <div class="row">
+      <div class="col-sm-offset-2 col-sm-8 card content-top-margin">
+        <div class="row tutorial-min">
+          <div class="col-sm-12">
+            <h2>Contact Us</h2>
+            <hr />
+            <p>
+              If you have any comments, questions or concerns, or if you'd like to reach us in order to get more involved, please feel free to reach out to us at <a href="#">godigitalwithus@gmail.com</a>.
```
Comment (dankozlowski) [inline app/views/static_pages/contact.html.erb]: Replace `<a href="#">godigitalwithus@gmail.com</a>` with an actual mailer anchor: `<a href="mailto:name@email.com">Link text</a>`

File: app/views/static_pages/contact.html.erb [hunk 2]
```diff
@@ -1,2 +1,17 @@
-<h1>StaticPages#contact</h1>
-<p>Find me in app/views/static_pages/contact.html.erb</p>
+<div class="content-wrap">
+  <div class="container footer-margin">
+    <div class="row">
+      <div class="col-sm-offset-2 col-sm-8 card content-top-margin">
+        <div class="row tutorial-min">
+          <div class="col-sm-12">
+            <h2>Contact Us</h2>
+            <hr />
```
Comment (dankozlowski) [inline app/views/static_pages/contact.html.erb]: Self-closing tags is an XML thing and not necessary in HTML5.  However, if this is a purposeful stylistic decision, self-close your `<br>`'s in the footer: `<br />

File: config/routes.rb [hunk 3]
```diff
@@ -1,60 +1,7 @@
 Rails.application.routes.draw do
   root 'static_pages#home'
   get 'guide' => 'static_pages#guide'
+  get 'contact' => 'static_pages#contact'
```
Comment (dankozlowski) [inline config/routes.rb]: Nit: I'm a complete nerd about this kinda crap.  Please line up the RHS and LHS of the hashrocket.

`Nit` indicates the suggested delta following is at your discretion ;)

**Comment by dankozlowski** [comment]
@lesliekimm CR complete.  Overall, good work, and feel free to ignore the nit comment.

**Comment by dankozlowski** [comment]
@lesliekimm More on self-closing tags: http://stackoverflow.com/questions/3558119/are-self-closing-tags-valid-in-html5

File: app/views/static_pages/contact.html.erb (hunk 1 above)
Comment (lesliekimm) [inline app/views/static_pages/contact.html.erb]: oh i was going to ask a question about this. so i personally HATE it when people use mailer anchors. is there any other solution?

Comment (dankozlowski) [inline app/views/static_pages/contact.html.erb]: You read me like a poem.  The answer is contact forms with a captcha.

Comment (dankozlowski) [inline app/views/static_pages/contact.html.erb]: Typically, the form will just fire off an email any way, but it decreases the amount of spam in two ways:

1. Captcha.  
2. No scrapable email address for bots just skimming the page.

Comment (lesliekimm) [inline app/views/static_pages/contact.html.erb]: okay great. will merge with mailer anchor for now and add contact forms with captcha to backlog :)

**Comment by lesliekimm** [comment]
thanks @dankozlowski!!!

**Comment by dankozlowski** [comment]
:+1:

(Typically, :+1:, or :shipit: for cool places like MS, means merge at will.)

**Comment by lesliekimm** [comment]
hahaha i like it! we use LGMT at lever

**Comment by dankozlowski** [comment]
@lesliekimm Stoked to be on this!  Let me know where I can help.

**Pull request merged**